panels/background/cc-background-xml.c

```diff
@@ -426,6 +429,33 @@ cc_background_xml_load_from_dir (const gchar      *path,
   g_object_unref (enumerator);
 }
 
+static gchar *
+read_vendor_background_directory (void)
+{
+  g_autoptr(GKeyFile) keyfile = NULL;
+  g_autoptr(GError) error = NULL;
+  gchar *result = NULL;
```

Comment:
Nitpick: `g_autofree gchar *result = NULL` then `return g_steal_pointer (&result);`

Reply:
Hmm, interesting, is there any benefit/difference of doing that? (just curious)